content/windows.md

```diff
@@ -28,15 +28,18 @@ section: software
 tableOfContents: true
 ---
 
-## Disclaimer
+## Important Disclaimer - Please Read
 
 System76 is not a licensed reseller or installer of the Windows operating system. This article is provided for information purposes only.
 
-System76 encourages users to take ownership of their machines and install whatever software or operating systems they prefer. However, System76 does not guarantee the success or quality of experience when installing Windows.
+System76 encourages users to take ownership of their machines and install whatever software or operating systems they prefer. However, System76 does not guarantee the success or quality of experience when installing Windows and does not provide support or testing for any Windows version. Please do not open a support ticket for anything other than clarification of the steps covered in this article.
+The contents of this support article are the **total extent of support and troubleshooting that System76 can provide for Windows.** Any troubleshooting or Windows support questions not covered in this article are outside the scope of System76 and should be referred to Microsoft.
```

Comment:
Consider changing `Any troubleshooting or Windows support questions not covered in this article are outside the scope of System76 and should be referred to Microsoft.`

to:

`Any Windows-related support for System76 hardware not covered in this article is beyond the scope of support provided by System76. Please refer to Microsoft's [official troubleshooting documentation](https://support.microsoft.com/en-us) or other online resources.`